dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java

```diff
@@ -42,94 +58,78 @@ public String getName() {
 
     @Override
     public String getHowTo() {
-        return "This actionlet will automatically publish the the content object to the specified enviroment(s). Multiple Environments can be separated by a comma";
+        return "This actionlet will automatically publish the the content object to the specified environment(s). Multiple Environments can be separated by a comma";
     }
 
-    public void executeAction(WorkflowProcessor processor, Map<String, WorkflowActionClassParameter> params)
+    @Override
+    public void executeAction(final WorkflowProcessor processor, final Map<String, WorkflowActionClassParameter> params)
             throws WorkflowActionFailureException {
+        final List<String> identifiers = new ArrayList<>();
+        final String environments = params.get(PARAM_ENVIRONMENT).getValue();
         try {
-            // Gets available languages
-            // List<Language> languages = languagesAPI.getLanguages();
-
-            Contentlet ref = processor.getContentlet();
-
-            boolean _contentPushNeverExpire = true;
-
-            boolean forcePush = ("true".equals(params.get("force").getValue())) ? true : false;
-            String envrions = params.get("environment").getValue();
-            if (envrions == null) {
-                Logger.error(this.getClass(), "There are no environments set to push to");
+            final Contentlet contentlet = processor.getContentlet();
+            final boolean forcePush = ("true".equals(params.get(PARAM_FORCE_PUSH).getValue())) ? Boolean.TRUE : Boolean.FALSE;
+            if (!UtilMethods.isSet(environments)) {
+                Logger.error(this, "There are no Push Publishing environments set to send the bundle.");
             }
-
-            String[] whereToSend = envrions.split(",");
-
-            List<Environment> envsToSendTo = new ArrayList<Environment>();
-            List<Environment> permissionedEnv = new ArrayList<Environment>();
-            List<Environment> finalEnvs = new ArrayList<Environment>();
-
-
+            final String[] whereToSend = environments.split(ENVIRONMENT_DELIMITER);
+            final List<Environment> envsToSendTo = new ArrayList<>();
+            final List<Environment> permissionedEnv = new ArrayList<>();
+            final List<Environment> finalEnvs = new ArrayList<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4986076)